src/i18n/de/nav.ts

```diff
@@ -1,94 +1,81 @@
-	'guides/prefetch': 'Vorabladen',
-	'guides/dev-toolbar': 'Dev Toolbar',
-
-	// Integrations
-	addons: 'Integrationen',
-	'guides/integrations-guide': 'Integrationen hinzufügen',
-	'guides/framework-components': 'UI-Frameworks',
-	'guides/server-side-rendering': 'Serverseitiges Rendern (SSR)',
-	'guides/astro-db': 'Astro DB',
-
-	// Recipes
-	examples: 'Anleitungen',
-	'guides/migrate-to-astro': 'Zu Astro migrieren',
+	content: 'Verwalte deine Inhalte',
+	'guides/content': 'Inhalte verfassen',
+	'guides/markdown-content': 'Markdown',
+	'guides/content-collections': 'Sammlungen von Inhalten',
 	'guides/cms': 'Ein CMS verbinden',
-	'guides/backend': 'Backend-Dienste hinzufügen',
-	'guides/deploy': 'Website veröffentlichen',
-	'guides/recipes': 'Mehr Anleitungen',
+	'guides/rss': 'Einen RSS-Feed hinzufügen',
 
-	// Guides
-	features: 'Themenbereiche',
+	routes: 'Routen und Navigation',
 	'guides/routing': 'Routing',
-	'guides/markdown-content': 'Markdown & MDX',
-	'guides/client-side-scripts': 'Skripte & Ereignisbehandlung',
+	'guides/endpoints': 'Endpunkte',
+	'guides/actions': 'Aktionen',
+	'guides/prefetch': 'Vorladen',
+	'guides/middleware': 'Middleware',
+	'guides/internationalization': 'Internationalisierung',
+	'guides/view-transitions': 'Übergänge von Ansichten',
+
+	assets: 'Ressourcen',
 	'guides/styling': 'CSS & Styling',
 	'guides/images': 'Bilder',
 	'guides/fonts': 'Schriftarten',
-	'guides/imports': 'Importe',
-	'guides/endpoints': 'Endpunkte',
-	'guides/data-fetching': 'Abrufen von Daten',
-	'guides/internationalization': 'Internationalisierung',
-	'guides/middleware': 'Middleware',
+	'guides/media': 'Hosted Media oder DAM verbinden',
+
+	data: 'Verbinde deine Daten',
+	'guides/data-fetching': 'Daten abrufen',
+	'guides/astro-db': 'Astro-DB',
+	'guides/backend': 'Backend-Dienste hinzufügen',
 	'guides/ecommerce': 'E-Commerce',
-	'guides/testing': 'Testen',
 	'guides/authentication': 'Authentifizierung',
-	'guides/troubleshooting': 'Fehlerbehebung',
-
-	// Configuration
-	configuration: 'Konfiguration',
-	'guides/configuring-astro': 'Die Astro-Konfigurationsdatei',
-	'guides/typescript': 'TypeScript',
-	'guides/aliases': 'Import-Aliasnamen',
 	'guides/environment-variables': 'Umgebungsvariablen',
 
-	// Reference
+	clientSide: 'Client-seitige Interaktivität',
+	'guides/framework-components': 'UI-Frameworks',
+	'guides/client-side-scripts': 'Skripte & Ereignisbehandlung',
+
+	maintain: 'Halte dein Projekt am Laufen',
+	'upgrade-astro': 'Astro aktualisieren',
+	'guides/testing': 'Tests',
+	'guides/troubleshooting': 'Fehlerbehebung',
+
 	reference: 'Referenz',
 	'reference/configuration-reference': 'Konfiguration',
-	'reference/api-reference': 'Laufzeit-API',
-	'reference/integrations-reference': 'Integrationen-API',
+	'reference/api-reference': 'Astro Laufzeit-API',
+	'reference/cli-reference': 'Astro-CLI',
+	'reference/directives-reference': 'Richtlinien-Referenz',
+	'reference/typescript': 'TypeScript-Referenz',
```

Comment:
Aus irgendeinem Grund wird sowohl diese Übersetzung als auch der Punkt "Built-in Components" in der Deploy Preview nicht übersetzt angezeigt. Bitte prüfe nochmal, ob der Key hierzu stimmt und ob ggf. die Übersetzung von "Built-in Components" fehlt.

Reply:
Okay, Built in components war auch ned in der französischen Übersetzung, obwohls auf i18n.docs.astro.build als completed angezeigt wird... Wurde dort wohl übersehen...

Reply:
Und der Key für die Typescript referenz ist einfach guides/typescript, auch falsch in der französischen Übersetzung...

Reply:
Ich habs nämlich von der französischen kopiert muss ich gestehen, weil sonst das kopieren der ganzen Keys aus der englischen Version aufwendig wäre, aber gut, dass du die Fehler bemerkt hast, danke! 😊

Reply:
Ok, top, vielen Dank! Ich gebe intern Bescheid, damit das auch in der französischen Übersetzung korrigiert wird.

Reply:
habs hier jetzt auf jeden Fall mal ausgebessert